<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>736</width>
    <height>266</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>0</y>
      <width>661</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>RACIONAIS</string>
    </property>
    <widget class="QLineEdit" name="lineEditDenominadorR">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>110</y>
       <width>61</width>
       <height>25</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditDenominador2">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>110</y>
       <width>61</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditDenominador1">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>110</y>
       <width>61</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>80</y>
       <width>21</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>=</string>
     </property>
    </widget>
    <widget class="Line" name="line_3">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>90</y>
       <width>118</width>
       <height>3</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditNumerador2">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>50</y>
       <width>61</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>90</y>
       <width>118</width>
       <height>3</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonOperacao">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>80</y>
       <width>41</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditNumeradorR">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>50</y>
       <width>61</width>
       <height>25</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>118</width>
       <height>3</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditNumerador1">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>50</y>
       <width>61</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonExecutar">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>50</y>
       <width>89</width>
       <height>90</height>
      </rect>
     </property>
     <property name="text">
      <string>EXEUTAR</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>736</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lineEditNumerador1</tabstop>
  <tabstop>lineEditDenominador1</tabstop>
  <tabstop>lineEditNumerador2</tabstop>
  <tabstop>lineEditDenominador2</tabstop>
  <tabstop>pushButtonExecutar</tabstop>
  <tabstop>lineEditNumeradorR</tabstop>
  <tabstop>lineEditDenominadorR</tabstop>
  <tabstop>pushButtonOperacao</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
